Tests/SonarScanner.MSBuild.PreProcessor.Test/SonarCloudWebServerTest.cs

```diff
@@ -229,9 +229,9 @@ public async Task DownloadCache_UserInputSupersedesAutomaticDetection(string var
         }
 
         [TestMethod]
-        [DataRow("http://cacheBaseUrl:222", "http://cacheBaseUrl:222/v1/sensor_cache/prepare_read?organization=org42&project=project-key&branch=project-branch")]
-        [DataRow("http://cacheBaseUrl:222/", "http://cacheBaseUrl:222/v1/sensor_cache/prepare_read?organization=org42&project=project-key&branch=project-branch")]
-        [DataRow("http://cacheBaseUrl:222/sonar/", "http://cacheBaseUrl:222/sonar/v1/sensor_cache/prepare_read?organization=org42&project=project-key&branch=project-branch")]
+        [DataRow("http://cacheBaseUrl:222", "http://cachebaseurl:222/v1/sensor_cache/prepare_read?organization=org42&project=project-key&branch=project-branch")]
+        [DataRow("http://cacheBaseUrl:222/", "http://cachebaseurl:222/v1/sensor_cache/prepare_read?organization=org42&project=project-key&branch=project-branch")]
+        [DataRow("http://cacheBaseUrl:222/sonar/", "http://cachebaseurl:222/sonar/v1/sensor_cache/prepare_read?organization=org42&project=project-key&branch=project-branch")]
```

Comment:
These strings go through the `new Uri()` constructor and become lowercase, and since case-ing does not give us any value here, i changed it to make the tests more concise, instead of doing `cacheFullUrl.ToString()` on the logger check in line 247.